modules/ossm-deploying-automatic-gateway-injection.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-traffic-manage.adoc
+
+:_content-type: PROCEDURE
+[id="ossm-deploying-automatic-gateway-injection_{context}"]
+= Deploying automatic gateway injection
+
+When deploying a gateway, you must opt-in to injection by adding an injection label or annotation to the gateway `deployment` object. The following example  `ConfigMap` object deploys a gateway with automatic injection.
+
+.Prerequisites
+
+* The namespace must be a member of the mesh (as defined in the `ServiceMeshMemberRoll` or by creating a `ServiceMeshMember` resource.
```

Comment:
```suggestion
* The namespace must be a member of the mesh by defining it in the `ServiceMeshMemberRoll` or by creating a `ServiceMeshMember` resource.
```
- Omit parenthesis. 
- Consider rewording to make the two phrases grammatically parallel.

Reply:
Done